VideoLocker/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -44,6 +44,10 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        final Bundle bundle = getArguments();
```

Comment:
As [mentioned](https://github.com/edx/edx-app-android/pull/408/files#r43965950) in #408, this variable should be renamed to "args" to be more descriptive.